src/components/MainTable/MainTable.js

```diff
@@ -141,6 +148,7 @@ const generateRows = (
 const MainTable = ({
   defaultSort,
   defaultSortDirection,
+  emptyStateMsg = "No data to display",
```

Comment:
I don't think this should have a default so that people can have no message if they want to.